flake.nix

```diff
@@ -2,43 +2,58 @@
   description = "dapptools";
 
   inputs = {
-    # same as in default.nix
-    nixpkgs.url = "github:NixOS/nixpkgs/aa576357673d609e618d87db43210e49d4bb1789";
+    nixpkgs.url = "github:NixOS/nixpkgs/2dea8991d89b9f1e78d874945f78ca15f6954289";
```

Comment:
Is there a reason to specify a pinned hash here? can't we just leave that to the `flake.lock`?

Reply:
Nope you're right, I'll fix this. I think the best way is to just use nixpkgs/master, and have a github action that periodically updates the flake.lock, so we catch issues early.